doc/classify.md

```diff
@@ -0,0 +1,100 @@
+# Tree-based classification
+
+Woltka features a highly flexible hierarchical classification system. It is represented by a **tree** structure instead of a fixed number of levels (e.g., the eight standard taxonomic ranks). In another word, it is **rank-free**.
+
+Wolkta supports various formats of classification systems, specifically:
+
+1. `--nodes`: NCBI-style `nodes.dmp` or a simple map, in which each taxon points to its parent taxon (2nd column). Rank (3rd column) is optional.
```

Comment:
I think we are still missing a rigorous definition of what a 'simple map' is.

Reply:
I added bit more description: "NCBI-style `nodes.dmp` (columns are delimited by "\<tab\>|\<tab\>") or a tab-delimited file, in which each taxon (1st column) points to its parent taxon (2nd column). Rank (3rd column) is optional."

The term "simple map" appears multiple times, and they refer to different things. The only common point is that they are all tab-delimited files. I added description to every instance where it occurs, as e.g., "a simple map of query <tab> subject". Commit [66cfa9c](https://github.com/qiyunzhu/woltka/pull/1/commits/66cfa9cf115dcdf548fc4ef2baab951442c8a4d9).